### DISCUSSION TITLE
刚刚委托的维多利亚家政的工作人员睡着了

### DISCUSSION THREAD

**Discussion by Loxic123** [share121/inter-knot#93] [其他]
维多利亚家政这么松懈吗，我现在要怎么把她叫醒，在线等，急！！！！
![20240711173644](https://github.com/user-attachments/assets/0b410bb5-8bba-4fa2-84fd-983d4a7d0960)

**Comment by Luli-Chen** [comment]
可以在旁边放一些棒棒糖

**Comment by Mu-7593** [comment]
试试在她耳边说：“ 再不起来我要摸你的尾巴喽 ”

**Comment by asdhvq** [comment]
只需拿根棒棒糖即可
